src/graphql-types/ReasonApolloMutation.re

```diff
@@ -2,7 +2,7 @@ open ReasonApolloTypes;
 
 module MutationFactory = (Config:Config) => {
     external cast : string => {. "data": Js.Json.t, "loading": bool} = "%identity";
-    [@bs.module] external gql : ReasonApolloTypes.gql = "graphql-tag";
+    [@bs.module "graphql-tag"] external gql : ReasonApolloTypes.gql = "default";
```

Comment:
This is the exact same thing: https://bucklescript.github.io/docs/en/import-export.html

Reply:
They seem to have updated the docs showing the difference: https://bucklescript.github.io/docs/en/import-export.html#import-an-es6-default-value